client/lib/invites/actions.js

```diff
@@ -124,8 +126,11 @@ export function sendInvites( siteId, usernamesOrEmails, role, message, formId )
 			siteId, usernamesOrEmails, role, message
 		} );
 		wpcom.undocumented().sendInvites( siteId, usernamesOrEmails, role, message, ( error, data ) => {
+			const validationErrors = get( data, 'errors' );
```

Comment:
To make sure we're on the same page, you're suggesting add the list of failed validations to `error`?

If so, I'm not sure how we would add an (object | array) of failed invitations to the error object since it expects a string key, string message, and number error code. We could stringify the object and add it to the message, but that feels awkward.